### PR TITLE
update Read me and fix plot bug

### PR DESCRIPTION
**Pull request by cbellot000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,27 @@
...
-files as well as several neutral formats (csv, hdf5, vtk,
-etc.). Various operators are available allowing the manipulation and
-the transformation of this data.
+transforming simulation data.
 
-The Python `ansys-dpf-post` package provides a simplified Python
-interface to DPF, thus enabling rapid postprocessing without ever
-leaving a Python environment. 
+The Python `ansys-dpf-post` package provides a high level, physics oriented API for postprocessing.
+Loading a simulation (defined by its result files) allows to extract simulation metadata as well
```
Comment (JennaPaikowsky) [inline README.md:12]: Either "allows you to extract simulation...." or "(defined by its result files) extracts simulation metadata....."

File: README.md [hunk 2]
```diff
@@ -42,58 +42,55 @@ cd pydpf-post
...
-### Running DPF-Post
-Provided you have ANSYS 2021R1 installed, a DPF server will start
-automatically once you start using DPF-Post.  Should you wish to use
-DPF-Post without 2020R1, see the [DPF Docker](https://dpfdocs.pyansys.com/getting_started/docker.html) documentation.
 
-Opening and plotting a result file generated from Ansys workbench or
-MAPDL is as easy as:
+## Brief Demo
+
+Provided you have ANSYS 2023R1 installed, a DPF server will start
```
Comment (JennaPaikowsky) [inline README.md:48]:
```suggestion
Provided you have ANSYS 2023 R1 installed, a DPF server will start
```

File: README.md [hunk 3]
```diff
@@ -42,58 +42,55 @@ cd pydpf-post
...
-The DPF-Post module is based on DPF Framework that been developed with
-a data framework that localizes the loading and post-processing within
-the DPF server, enabling rapid post-processing workflows as this is
-written in C and FORTRAN.  At the same time, the DPF-Post Python
-module presents the result in Pythonic manner, allowing for the rapid
-development of simple or complex post-processing scripts.
-
-
-**Easy to use**
+To use PyDPF-Post, with Ansys versions starting from 2021R1 to 2022R2, leverage legacy PyDPF-Post
```
Comment (JennaPaikowsky) [inline README.md:80]:
```suggestion
To run PyDPF-Post with Ansys versions starting from 2021 R1 to 2022 R2, use the following legacy PyDPF-Post 
```

**Review by JennaPaikowsky — COMMENTED** [review]
Just a few comments for this PR

File: README.md (hunk 1 above)
Comment (cbellot000) [reply README.md:12]: Thanks @JennaPaikowsky I'll go with the first option as it's not loading the simulation that extracts but it's something one can after loading the simulation
```suggestion
Loading a simulation (defined by its result files) allows you to extract simulation metadata as well
```

**Pull request merged**